main/src/library/Array.flix

```diff
@@ -76,13 +76,15 @@ mod Array {
         $ARRAY_STORE$(a, i, x)
 
     ///
-    /// Optionally returns the element at position `i` in the array `a`.
+    /// Returns the element at position `n` in `a`.
     ///
-    pub def nth(i: Int32, a: Array[a, r]): Option[a] \ r =
-        if (0 <= i and i < length(a))
-            Some(get(i, a))
+    /// If `n` is negative or `a` has length less than n, an IndexOutOfBounds effect is thrown.
+    ///
+    pub def nth(n: Int32, a: Array[a, r]): a \ {r, IndexOutOfBounds} =
```

Comment:
Note this is a breaking API change, which is causing one of the community builds to fail.

Reply:
Ok